integration-authentication-api/pom.xml

```diff
@@ -16,6 +16,18 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
```

Comment:
I see. You need this dependency to map the JSON property name to class attributes.

I think we can use long rather than Date to simplify the code.